mineral-ui/mineral-ui
# [Website] content clarification; a11y updates

connects #427 

### Description
- expand on RTL acronyms
- add Example h2 for a11y reasons
- change first example title to be descriptive
- link-ify Icon category titles
- add to first Link description to explain why there's a discrete Link component at all
- add a bit of explanation about title vs alt attributes on an Icon
- fix some broken links

### Motivation and context
tightening site content and improving a11y.

https://expand-rtl--mineral-ui.netlify.com/

### How to test


1. `cd mineral-ui && npm start`
2. `npm run build:icons`
3. Navigate to each component page. There should be an H2 "Examples" followed by a demo with a more descriptive title.
4. Icon category links should be available
5. does the copy on the Icon `title` attribute example make sense?
6. does the copy on the first Link example make sense?

### Types of changes
- docs updates

### Checklist




### How does this PR make you feel?
![time to clean up](https://media.giphy.com/media/NMwqTq1XznkGI/giphy.gif)
